### PR TITLE
Don't force "the" article for holy books

### LINKED ISSUE #144: "The" holy book
In the current master branch, holy books are always referred to using the definite article "the". I believe using the indefinite article "a" would be more appropriate since there can be many holy books for each god. Some examples:

screen shot 2016-06-26 at 13 05 05

screen shot 2016-06-26 at 12 59 11

### PR DESCRIPTION
**Pull request by emlai**
Fixes #144.

### CODE REVIEW

File: Script/item.dat [hunk 1]
```diff
@@ -3844,4 +3843,3 @@ firstbornchild
     NameSingular = "daughter";
   }
 }
-
```
Comment (AlexMooney) [inline Script/item.dat]: It's usually best to leave unrelated lines alone.

**Comment by AlexMooney** [comment]
Won't we still end up with "The 2 holy banana of Oily Orpiv"?  I think it would be better to work on the `FORCE_THE` logic.

**Comment by AlexMooney** [comment]
Ah, never mind!  I see that the other `FORCE_THE` items don't seem to stack.  :+1:

File: Script/item.dat (hunk 1 above)
Comment (emlai) [inline Script/item.dat]: Yeah I know, however that instance didn't add any significant noise in the diff or blame, so I decided it wasn't worth it to leave it alone :)
